elixir/apps/web/lib/web/live/sites/edit.ex

```diff
@@ -28,12 +28,50 @@ defmodule Web.Sites.Edit do
           <.form for={@form} phx-change={:change} phx-submit={:submit}>
             <div class="grid gap-4 mb-4 sm:grid-cols-1 sm:gap-6 sm:mb-6">
               <div>
-                <.input
-                  label="Name Prefix"
-                  field={@form[:name]}
-                  placeholder="Name of this Site"
-                  required
-                />
+                <.input label="Name" field={@form[:name]} placeholder="Name of this Site" required />
+              </div>
+              <div>
+                <p class="text-lg text-slate-900 mb-2">
+                  Data Routing -
+                  <a
+                    class={[link_style(), "text-sm"]}
+                    href="https://www.firezone.dev/kb"
```

Comment:
```suggestion
                    href="https://www.firezone.dev/kb?utm_source=product"
```

This will help us organize analytics later on

Reply:
👍